app/mailers/user_mailer.rb

```diff
@@ -14,7 +14,28 @@ def self.send_team_leader_reminder
   def team_leader_reminder(recipient)
     @recipient = recipient
 
-    email_with_name = %("#{@recipient.fullname}" <#{@recipient.email}>)
-    mail(to: email_with_name, subject: "Monthly team members' initiatives reminder")
+    mail(to: email_with_name(recipient), subject: "Monthly team members' initiatives reminder")
+  end
+  
+  def self.send_team_member_reminder
+    @recipients = User.all
+    
+    @recipients.each do |recipient|
+      next unless recipient.initiatives.incompleted.any?
```

Comment:
What happens here if the recipient doesn't have any initiatives? Admin user, for example. Does it break?

Reply:
If a user don't have any initiative, mail will not send to the user. I have tested it.